manifests/role/kubernetes/backup_gateway.pp

```diff
@@ -11,5 +11,12 @@
   include nebula::profile::kubernetes::router
   include nebula::profile::kubernetes::bootstrap::source
   include nebula::profile::kubernetes::etcdctl
-  include nebula::profile::kubernetes::keepalived
+
+  class { 'nebula::profile::kubernetes::keepalived':
+    master => false,
+  }
```

Comment:
This change is just to make the backup/primary role file diffs easier to see at a glance.

```diff
 class { 'nebula::profile::kubernetes::keepalived':
-  master => false,
+  master => true,
 }
```

instead of

```diff
-include nebula::profile::kubernetes::keepalived
+class { 'nebula::profile::kubernetes::keepalived':
+  master => true,
+}
```

No change in behavior.